### ISSUE TITLE
[Suggestion] Explanation for CSV format

### ISSUE THREAD

**Issue by jbt-cii** [aissat/easy_localization_loader#22]
Hello,

Thank you for your package, it is very helpful.

I make a suggestion: it would be nice to explain a little bit some things about the CSV format (and maybe for others formats).

I made several tests to understand what it was possible to do with CSV format.

So I know now that CSV works only:

- with only comma separator (semicolon or other separator are not accepted)

- with a specific header : str,en_US,fr_FR, and so on

- with or without double quotes

- the key can be a keyword or simply the english version... we can mix the 2 ways, for instance:

"str","en_US","fr_FR"
"button to change the language into french","button to change the language into french","bouton pour passer l'application en langue anglaise"
"form.error.client_user.not_valid","please enter a valid user","merci de saisir un utilisateur valide"

- and when we change a translation, we need to rebuild the whole application

Personally, with each modification, I redo a sort on the CSV file according to the key to reorder everything in alphabetical order to identify possible duplicates. 
Do not forget to put the header back in the 1st line afterwards.

**Comment by SugaR256** [comment]
I agree, at least a CRLF instead of LF warning would be appreciated.

**Comment by ben-xD** [comment]
More information is always welcome, but I found that the example app has some files, including a [csv](https://github.com/aissat/easy_localization/blob/master/example/resources/langs/langs.csv). 

Folder: https://github.com/aissat/easy_localization/tree/master/example/resources/langs
